CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 
+<a name="0.4.0"></a>
+## [0.4.0](https://github.com/t4d-gmbh/stubbed_versioning/compare/0.4.0-rc...0.4.0)
```

Comment:
Several release candidates pick up the tag from the previous release candidate.

What should happen is that the last "clean' tag is picked for comparison.